### PR TITLE
Akka 2.5.22 released

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: blog/_posts/2019-04-03-akka-2.5.22-released.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+     18   2977      430 Christopher Batey
+     13    952      869 Nicolas Vollmar
+     12    837      271 Helena Edelson
+      7     13        7 Yoel Garcia Diaz
+      6    467       67 Luc Bourlier
+      5    172       68 Gaël Bréard
+      5     26      147 Johannes Rudolph
+      5     26       22 Enno
+      4     19        6 Varun Sivapalan
+      2  35013    24576 Auto Format
```
Comment (johanandren) [inline blog/_posts/2019-04-03-akka-2.5.22-released.md]: Maybe mr Auto Format shouldn't be in here

File: blog/_posts/2019-04-03-akka-2.5.22-released.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  supervision is based on the default behavior of the child i.e. if an untyped actor creates a typed
+  child, its default supervision will be to stop. If a typed actor creates an untyped child, its
+  default supervision will be to restart.
+* Fix memory leak in `watchWith`, thanks to [@longshorej](https://github.com/longshorej), [#26625](https://github.com/akka/akka/issues/26625)
+* Prevent surprising ordering of `unstashAll` and side effects in `EventSourcedBehavior` by introducing `EffectBuilder` type, [#26489](https://github.com/akka/akka/issues/26489)
+* Use signals instead of callbacks in `EventSourcedBehavior`, [#25428](https://github.com/akka/akka/issues/25428)
+* Change signature of `terminate`, `whenTerminated` and `getWhenTerminated` in `ActorSystem`, [#25647](https://github.com/akka/akka/issues/25647)
+* New classes for `SnapshotSelectionCriteria` and `SnapshotMetadata`, [#26536](https://github.com/akka/akka/issues/26536)
+* Fix leak of timers when stopping, [#26285](https://github.com/akka/akka/issues/26285)
+* Several improvements of stashing
```
Comment (viktorklang) [inline blog/_posts/2019-04-03-akka-2.5.22-released.md]: Perhaps elaborate a bit on what kind of improvements these are: correctness, performance, memory usage, or other?

Comment (patriknw) [reply blog/_posts/2019-04-03-akka-2.5.22-released.md]: fixed

File: blog/_posts/2019-04-03-akka-2.5.22-released.md (hunk 1 above)
Comment (patriknw) [reply blog/_posts/2019-04-03-akka-2.5.22-released.md]: ok, removed

File: blog/_posts/2019-04-03-akka-2.5.22-released.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+After some final scalability testing we can now declare [Artery TCP](https://doc.akka.io/docs/akka/current/remoting-artery.html)
+ready to be used in production. In Akka 2.6.0 we will make the Artery TCP transport the default for actor messaging.
+See the short [migration guide](https://doc.akka.io/docs/akka/current/remoting-artery.html#migrating-from-classic-remoting)
+for how to switch.
+
+Artery improves stability for systems using Akka Cluster and also has much better performance.
+
+One thing to be aware of is that rolling update from classic remoting to Artery is not supported since the protocol
+is completely different. It will require a full cluster shutdown and new startup.
```
Comment (raboof) [inline blog/_posts/2019-04-03-akka-2.5.22-released.md]: Maybe clarify classic remoting isn't going away for a while?

Comment (patriknw) [reply blog/_posts/2019-04-03-akka-2.5.22-released.md]: added a sentence

**Pull request merged**